packages/next/README.md

```diff
@@ -1701,26 +1699,24 @@ export default Error
 If you want to render the built-in error page you can by using `next/error`:
 
 ```jsx
-import React from 'react'
-import Error from 'next/error'
-import fetch from 'isomorphic-unfetch'
-
-class Page extends React.Component {
-  static async getInitialProps() {
-    const res = await fetch('https://api.github.com/repos/zeit/next.js')
-    const errorCode = res.statusCode > 200 ? res.statusCode : false
-    const json = await res.json()
+import React from "react"
+import Error from "next/error"
+import fetch from "isomorphic-unfetch"
 
-    return { errorCode, stars: json.stargazers_count }
+const Page = ({ errorCode, stars }) => {
+  if (errorCode) {
+    return <Error statusCode={errorCode} />
   }
 
-  render() {
-    if (this.props.errorCode) {
-      return <Error statusCode={this.props.errorCode} />
-    }
+  return <div>Next stars: {stars}</div>
+}
 
-    return <div>Next stars: {this.props.stars}</div>
-  }
+Page.getInitialProps = async () => {
+  const res = await fetch("https://api.github.com/repos/zeit/next.js")
```

Comment:
It's minor, but it looks like the README currently uses single quotes for strings.